src/main/java/nl/tudelft/jpacman/level/DefaultPlayerInteractionMap.java

```diff
@@ -16,6 +17,18 @@
  */
 public class DefaultPlayerInteractionMap implements CollisionMap {
 
+    private PointCalculator pointCalculator;
+
+    /**
+     * Create a simple player-based collision map, informing the
+     * point calculator about points to be added.
+     * @param pointCalculator Strategy for calculating points.
```

Comment:
For consistency, in all other parts of the code base there's an empty line between the description and `@param` (holds for other files in this PR as well ;) )

Reply:
Done!